E2a-TRAS/PJ01/README.md

```diff
@@ -0,0 +1,160 @@
+
+
+
+
+## Expected Results
+
+For certification of capabilities the only source will be:
+
+- n. 1 IFC file containing the information as requested. The file shall be named using the following syntax: `MVDCode`_`ExchangeCode`_`TestCode`_`SoftwareVendor`.`ifc` (Example: `IFC4.3AbRV_E2a_PJ01_AmazingSoft.ifc`)
+
+
+
+
+## Validation criteria
+:zap: For this test case to be considered passed **all capabilities listed in this section** shall be verified, with no exception. :zap:
+
+### General
+
+- All the concept templates must be correctly implemented as presented in the validation criteria
+- At least 1 instance of each entity listed in [Itemised Roots](#Itemised-Roots) is present in the file
+
+
+### Project Setup
+
+> **Acceptance criteria**
+ For the **Project Setup** capability, the validation procedure must verify that:
+> - `IfcProject` entity is the top most element in the project structure. It represents the bare minimum of an IFC file exchange with no content, only boilerplate code.
+> - `IfcUnitAssignment` entity is used to define the global units for measures and values, when the units are not otherwise defined. For the specific units required by this test refer to the table below.
+> - `IfcGeometricRepresentationContext` entity is used to define the is the global context of the geometry.
+>
+> If present, all criteria listed in [Usages, Constraints & Logic](#Usages,-Constraints-&-Logic), and in the same section of precondition tests, shall be verified too.
+
+
+| **ID**  | **CRITERIA**                                      | **VALUE**                           | **COMMENT**                           |
+|---------|---------------------------------------------------|-------------------------------------|---------------------------------------|
+| ENAT_01 | Requested entities (and attributes) exist in file | See below                           |                                       |
+| ORIG_01 | Origin of Coordinate System is set as requested   | (0., 0., 0.)                        |                                       |
+| ORIG_02 | True north is set as requested                    | (0., 1., 0.)                        | Positive Y axis = geographic northing |
+| DIST_01 | Unit of measure for all distances                 | meter                               |                                       |
+| ANGL_01 | Unit of measure all angles                        | radian                              |                                       |
+| DIST_02 | Required precision for **distances**              | minimum 4 decimal places (0,0001)   |                                       |
+| ANGL_02 | Required precision for **angles** and **slope**   | minimum 6 decimal places (0,000001) |                                       |
```

Comment:
I'm unsure where is this saved in the IFC file. Where do you expect to find these values?

Reply:
For alignment, almost everywhere. Example in Storyline we checked that (e.g. for Horizontal alignment) Start Point X, Start Point Y, Segment Length had > 4 dec places, and Start Direction had > 6